content/departments/engineering/product/process/index.md

```diff
@@ -38,6 +38,30 @@ See [strategy](../../../../strategy-goals/index.md).
 - [Pricing and tiering/packaging](gtm/pricing.md)
 - [Licensing](gtm/licensing.md)
 
+#### Experimenting on sourcegraph.com
+
+We welcome experimentations on sourcegraph.com. It should be very easy to experiment on .com, send a tweet or email campaign out to get some traffics, and get early feedback in.
+
+The bar for an experiment on .com is the following:
+
+- Break nothing
```

Comment:
(new UIs and public data APIs are low risk, anything involving sensitive data or critical APIs is high risk)